### PR TITLE
Ensure turbine api calls to turbine grpc

### PR DESCRIPTION
**Pull request by dianadoherty**
## Description of change

Ensure turbine api calls to turbine grpc

Fixes https://github.com/meroxa/turbine-go/issues/137

## Type of change

- [x]  New feature
- [ ]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## How was this tested?

- [ ]  Unit Tests
- [ ]  Tested in staging
- [ ]  Tested in minikube

## Demo

| before | after |
|--------|-------|
|||

### CODE REVIEW

File: pkg/app/record.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	"time"
+
+	"github.com/meroxa/turbine-go/pkg/proto/core"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type Records struct {
+	Stream  string
+	Records []Record
+	Name    string
```
Comment (lyuboxa) [inline pkg/app/record.go:13]: I know this is silly, but do you mind moving name to the top of the struct ;-)

File: pkg/app/record.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+func NewRecords(c *core.Collection) Records {
+	rs := []Record{}
+	for _, r := range c.Records {
+		rs = append(rs,
+			Record{
+				Key:       r.Key,
+				Payload:   r.Value,
+				Timestamp: r.Timestamp.AsTime(),
```
Comment (lyuboxa) [inline pkg/app/record.go:23]: Should we have this always as UTC?

File: pkg/app/record.go [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+			})
+	}
+	return &core.Collection{
+		Stream:  rs.Stream,
+		Records: rds,
+		Name:    rs.Name,
+	}
+}
+
+type Record struct {
```
Comment (lyuboxa) [inline pkg/app/record.go:52]: Could we move the type/struct definitions to the top?

File: pkg/turbine/process.go [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+			Collection: rs.ToProto(),
+		})
+	if err != nil {
+		return app.Records{}, err
+	}
+
+	records := app.NewRecords(c)
+
+	rawOut := fn.Process(records.Records)
+	records.Records = rawOut
```
Comment (lyuboxa) [inline pkg/turbine/process.go:32]: hrm, do we want to dup records? or is this a ptr?

Comment (lyuboxa) [reply pkg/turbine/process.go:32]: resolving this, its a pass by value so we are ok.

**Review by lyuboxa — APPROVED** [review]

File: pkg/app/record.go (hunk 3 above)
Comment (dianadoherty) [reply pkg/app/record.go]: https://github.com/meroxa/turbine-go/pull/140/commits/f835039c2366b47be31b00b567c92dc511fe92f5

**Review by anna-cross — APPROVED** [review]

File: pkg/app/record.go (hunk 2 above)
Comment (lyuboxa) [reply pkg/app/record.go:31]: Ah thanks I missed this.

**Pull request merged**